packages/grafana-ui/src/components/Table/ImageCell.tsx

```diff
@@ -12,18 +14,23 @@ export const ImageCell: FC<TableCellProps> = (props) => {
   const displayValue = field.display!(cell.value);
 
   const hasLinks = Boolean(getCellLinks(field, row)?.length);
+  const clearButtonStyle = useStyles2(clearLinkButtonStyles);
 
   return (
     <div {...cellProps} className={tableStyles.cellContainer}>
       {!hasLinks && <img src={displayValue.text} className={tableStyles.imageCell} alt="" />}
       {hasLinks && (
         <DataLinksContextMenu style={{ height: '100%' }} links={() => getCellLinks(field, row) || []}>
           {(api) => {
-            return (
-              <div onClick={api.openMenu} className={cx(tableStyles.imageCellLink, api.targetClassName)}>
-                <img src={displayValue.text} className={tableStyles.imageCell} alt="" />
-              </div>
-            );
+            if (api.openMenu) {
+              return (
+                <Button className={cx(clearButtonStyle)} onClick={api.openMenu}>
+                  <img src={displayValue.text} className={tableStyles.imageCell} alt="" />
```

Comment:
Do we need a more descriptive text here for the `alt` prop for better accessibility?

Reply:
I don't think we can set something useful for the alt text here as the image url is set from the value of the field. The only thing we could set it to would be the url, or maybe a substring of the url. Probably not something that would be very helpful to someone useing a screen reader.